easybuild/easyconfigs/r/R/R-3.4.4-foss-2018a-X11-20180131.eb

```diff
@@ -0,0 +1,1811 @@
+    }),
+    ('sp', '1.2-7', {
+        'checksums': ['6d60e03e1abd30a7d4afe547d157ce3dd7a8c166fc5e407fd6d62ae99ff30460'],
+    }),
+    ('zoo', '1.8-1', {
+        'checksums': ['d93d88500374fce7f5c047f25240ac60483bcfd6fadf81df4eb2b879ad2ccc9a'],
+    }),
+    ('lmtest', '0.9-35', {
+        'checksums': ['2f4982f1b6cd133ef36fe9718f4ff2f4fa5c20716100fdd5ee5c947b68c8eb80'],
+    }),
+    ('vcd', '1.4-4', {
+        'checksums': ['a561adf120b5ce41b66e0c0c321542fcddc772eb12b3d7020d86e9cd014ce9d2'],
+    }),
+    ('snowfall', '1.84-6.1', {
+        'checksums': ['5c446df3a931e522a8b138cf1fb7ca5815cc82fcf486dbac964dcbc0690e248d'],
+    }),
+    ('rpart', '4.1-13', {
+        'checksums': ['8e11a6552224e0fbe23a85aba95acd21a0889a3fe48277f3d345de3147c7494c'],
+    }),
+    ('survival', '2.41-3', {
+        'checksums': ['f3797c344de93abd2ba8c89568770a13524a8b2694144ae55adec46921c8961d'],
+    }),
+    ('mice', '2.46.0', {
+        'checksums': ['5d3109ab3e9a84cb9ba60b0ab477d4e3ed9326fe4831fd99bbf1ed7fcca45bbe'],
+    }),
+    ('urca', '1.3-0', {
+        'checksums': ['621cc82398e25b58b4a16edf000ed0a1484d9a0bc458f734e97b6f371cc76aaa'],
+    }),
+    ('fracdiff', '1.4-2', {
+        'checksums': ['983781cedc2b4e3ba9fa020213957d5133ae9cd6710bc61d6225728e2f6e850e'],
+    }),
+    ('logistf', '1.22', {
+        'checksums': ['e866f02b1b40332e6e57c932201993725742e464f362e0053776bb8ce8c2fc5a'],
+    }),
+    ('akima', '0.6-2', {
+        'checksums': ['61da3e556553eea6d1f8db7c92218254441da31e365bdef82dfe5da188cc97ce'],
+    }),
+    ('bitops', '1.0-6', {
+        'checksums': ['9b731397b7166dd54941fb0d2eac6df60c7a483b2e790f7eb15b4d7b79c9d69c'],
+    }),
+    ('boot', '1.3-20', {
+        'checksums': ['adcb90b72409705e3f9c69ea6c15673dcb649b464fed06723fe0930beac5212a'],
+    }),
+    ('mixtools', '1.1.0', {
+        'checksums': ['543fd8d8dc8d4b6079ebf491cf97f27d6225e1a6e65d8fd48553ada23ba88d8f'],
+    }),
+    ('cluster', '2.0.6', {
+        'checksums': ['b53bb0eed64b79767a6e0d203fdc82f9fdb5c9692f30316fbe1c5b196a62fd81'],
+    }),
+    ('gclus', '1.3.1', {
+        'checksums': ['d6994ee264b464503db005450153973d75eba502e4caaaa6ff99cb95e4376a09'],
+    }),
+    ('coda', '0.19-1', {
+        'checksums': ['d41ff5731da6805170769dba75dd011ab33f916d15b2336001f279e21a524491'],
+    }),
+    ('codetools', '0.2-15', {
+        'checksums': ['4e0798ed79281a614f8cdd199e25f2c1bd8f35ecec902b03016544bd7795fa40'],
+    }),
+    ('foreach', '1.4.4', {
+        'checksums': ['c0a71090d5b70b9a95a6936091dabae9c26e1fc6b9609bfe5fb6346033905e48'],
+    }),
+    ('doMC', '1.3.5', {
+        'checksums': ['792254596babd8cc074bb3c83c18672dd07a0c3246b4ebab08d8ebedf6f4d9ed'],
+    }),
+    ('DBI', '0.8', {
+        'checksums': ['d52b9e04b3291f9bda8a88c174678f520ddffc5f2edf9b3dfa6d97dca943ce9a'],
+    }),
+    ('foreign', '0.8-69', {
+        'checksums': ['820e1409162fbd6418b80a0f5a4b50d649ee74f06d195a5210745619b16e1592'],
```

Comment:
This should be `13689f5ec1ab09e8973be81c7f1799b7de4313176072887a9fa0b5825aed3468`, see https://github.com/easybuilders/easybuild-easyconfigs/pull/6245